src/Config.js

```diff
@@ -89,6 +90,11 @@ module.exports = class extends ChainedMap {
           }
 
           case 'entry': {
+            if (typeof value === 'string') {
```

Comment:
Can you provide an example of what the intent is with this? Given a string value of entry, what entries does it generate in the config?